src/Hl7.Fhir.R4/Model/Generated/Questionnaire.cs

```diff
@@ -84,6 +84,12 @@ public enum QuestionnaireItemType
       [EnumLiteral("display"), Description("Display")]
       Display,
       /// <summary>
+      /// An item that defines a specific answer to be captured, and which may have child items. (the answer provided in the QuestionnaireResponse should be of the defined datatype).
```

Comment:
You cannot manually add code to generated files, these will be overwritten next time. 

Also, this code is no longer generated on purpose (it was accidentally generated, but we could never remove it until a major new release). This code is *abstract* (It's in italics in https://hl7.org/fhir/R4/valueset-item-type.html) and will never appear in an instance, only the codes below it in hierarchy, so it should not be part of this enum.

Your PR has no context or description on why you are doing this, so did you encounter this code somewhere? Did someone report a bug?